src/ShallowWrapper.js

```diff
@@ -80,8 +80,16 @@ export default class ShallowWrapper {
           }
         });
       });
-      this.node = this.renderer.getRenderOutput();
-      this.nodes = [this.node];
+      Object.defineProperty(this, 'node', {
+        get() {
```

Comment:
Yes, absolutely it would (when it's not fixing a bug, which admittedly can be a grey area)

If we want to be able to change things without it being a breaking change, then those things have to be *impossible to reach by consumers*, even by reflection.